sdks/typescript/featurehub-javascript-client-sdk/README.md

```diff
@@ -254,16 +256,46 @@ this.initializeFeatureHub();
 
 
 #### Supported feature state requests
+   
+  It is recommended if you want the raw feature (without any user context), you use the `feature` method of the 
+  repository. The repository is the in memory location where the contents of the FeatureHub server are deposited and
+  that your code interacts with. Each request for features does _not_ go back to the FeatureHub server, the repository
+  is kept up to date either by polling periodically (see polling) or via the near-realtime SSE connector.
```

Comment:
either by polling client or SSE mechanism (see above for Options to get feature updates)